azure-iot-device/azure/iot/device/iothub/sync_handler_manager.py

```diff
@@ -0,0 +1,232 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module contains the manager for handler methods used by the callback client"""
+
+import logging
+import threading
+import abc
+import six
+from azure.iot.device.common import handle_exceptions
+from azure.iot.device.common.chainable_exception import ChainableException
+from azure.iot.device.iothub.sync_inbox import InboxEmpty
+import concurrent.futures
+
+logger = logging.getLogger(__name__)
+
+MESSAGE = "_on_message_received"
+METHOD = "_on_method_request_received"
+TWIN_DP_PATCH = "_on_twin_desired_properties_patch_received"
+# TODO: add more for "event"
+
+
+class HandlerManagerException(ChainableException):
```

Comment:
>HandlerManagerException [](start = 6, length = 23)

add doc please and pass a message into the base constructor to explain what this signifies.

Reply:
I believe the message will automatically go into the base constructor due to inheritance. However, yes I will add some doc.

---
In reply to: [473407912](https://github.com/Azure/azure-iot-sdk-python/pull/644#discussion_r473407912) [](ancestors = 473407912)